client/app/components/composites/ListingCard/ListingCard.story.js

```diff
@@ -107,7 +119,7 @@ const testPrice = function priceTest(card, mountedCard) {
 };
 const testDistance = function priceTest(card, mountedCard) {
   it('Should display formatted distance', () => {
-    expect(mountedCard.text()).to.include(formatDistance(card.props.listing.distance, card.props.listing.distanceUnit));
+    expect(mountedCard.text()).to.include(formatDistance(card.props.listing.distance));
```

Comment:
Maybe also a test for currency formatting? E.g. locale specific separators and positioning, weird amount of fractionals